staff/manuel-eranueva/strings/concate copy.js

```diff
@@ -0,0 +1,17 @@
+delete String.prototype.concat
+
+function concat(string1, string2) {
+    var string3 = ''
+    if (string1 == Number && string2 == Number) {
+        string3 = string1.toString() + string2
+    } else {
+        string3 = string1 + string2
+    }
+    return string3
+}
+
+var string1 = 555
+var string2 = 5552
+
+var string3 = concat(string1, string2);
+console.log(string3);
```

Comment:
What happens ir we want to concatenate more than 2 elements? Check the concat() behaviour when we use it like this example:

string1.concat(string2, " ", string1, " ",string2)